jmix-flowui/flowui/src/main/java/io/jmix/flowui/UiEventPublisher.java

```diff
@@ -163,6 +167,28 @@ public void publishEventForUsers(ApplicationEvent event, @Nullable Collection<St
         clusterApplicationEventPublisher.publish(uiUserEvent);
     }
 
+    @Nullable
+    protected Collection<String> addCurrentUserIfNecessary(@Nullable Collection<String> usernames) {
```

Comment:
Let's add detailed JavaDoc that describes why we need this

Reply:
Fixed